src/test/scala/com/campudus/tableaux/ChangeTest.scala

```diff
@@ -0,0 +1,70 @@
+package com.campudus.tableaux
+
+import org.junit.Test
+import org.vertx.testtools.VertxAssert._
+import org.vertx.scala.core.json.{ Json, JsonObject }
+
+class ChangeTest extends TableauxTestBase {
+
+  val expectedJson = Json.obj("status" -> "ok")
+
+  @Test
+  def changeTableName(): Unit = okTest {
+    val postJson = Json.obj("tableName" -> "New testname")
+    val expectedString = "New testname"
+
+    for {
+      _ <- setupDefaultTable()
+      test <- sendRequestWithJson("POST", postJson, "/tables/1")
+      test2 <- sendRequest("GET", "/tables/1")
+    } yield {
+      assertEquals(expectedJson, test)
+      assertEquals(expectedString, test2.getString("tableName"))
+    }
+  }
+
+  @Test
+  def changeColumnName(): Unit = okTest {
+    val postJson = Json.obj("columns" -> Json.arr(Json.obj("name" -> "New testname")))
+    val expectedString = "New testname"
+
+    for {
+      _ <- setupDefaultTable()
+      test <- sendRequestWithJson("POST", postJson, "/tables/1/columns/1")
+      test2 <- sendRequest("GET", "/tables/1/columns/1")
+    } yield {
+      assertEquals(expectedJson, test)
+      assertEquals(expectedString, test2.getArray("columns").get[JsonObject](0).getString("name"))
+    }
+  }
+
+  @Test
+  def changeColumnOrdering(): Unit = okTest {
+    val postJson = Json.obj("columns" -> Json.arr(Json.obj("ordering" -> 5)))

```

Comment:
You are posting to a specific column. Why do you need to wrap the `"ordering" : 5`? (Like above)